src/Rings/mpoly-localizations.jl

```diff
@@ -765,13 +836,12 @@ inverted_set(W::MPolyLocalizedRing) = W.S
 gens(W::MPolyLocalizedRing) = W.(gens(base_ring(W)))
 
 ### required extension of the localization function
-Localization(S::MPolyComplementOfPrimeIdeal) = MPolyLocalizedRing(ambient_ring(S), S)
+Localization(S::AbsMPolyMultSet) = MPolyLocalizedRing(ambient_ring(S), S)
 
-Localization(S::MPolyComplementOfKPointIdeal) = MPolyLocalizedRing(ambient_ring(S), S)
-
-Localization(S::MPolyPowersOfElement) = MPolyLocalizedRing(ambient_ring(S), S)
-
-Localization(S::MPolyProductOfMultSets) = MPolyLocalizedRing(ambient_ring(S), S)
```

Comment:
Why remove these lines?

Reply:
I add a `Localization(::AbsMPolyMultSet)` since it is the same for all subtypes of this.

Reply:
Ok, that seems fair.